Xabe.FFmpeg/FFmpegWrapper.cs

```diff
@@ -106,7 +106,17 @@ internal class FFmpegWrapper : FFmpeg
                             return false;
                         }
 
-                        if (_outputLog.Any(x => x.Contains("Unknown decoder")))
+                        if (_outputLog.Any(x => x.Contains("Invalid NAL unit size")))
```

Comment:
Please extract all these strings to const array and check in loop if output string contains any of these errors.